estela-api/api/views/project.py

```diff
@@ -93,6 +101,11 @@ def update(self, request, *args, **kwargs):
                         )
                     elif action == "remove":
                         instance.users.remove(user)
+                    elif action == "update":
+                        instance.users.remove(user)
+                        instance.users.add(
+                            user, through_defaults={"permission": permission}
+                        )
```

Comment:
Why we add user instead of update user permissions?

Reply:
Because in Python sets, the update() function only add elements. If I update a user with other permission it would create another user with a different permission
https://www.programiz.com/python-programming/methods/set/update